libs/blocks/modal/decorators/langnav.js

```diff
@@ -0,0 +1,37 @@
+import { getConfig } from '../../../utils/utils.js';
+
+export async function handleEvent(prefix, link, config) {
```

Comment:
Does the fetch need to be returned?

`async` can be removed, or the fetch.then switched to using `await`

Reply:
yes, true. Will change

Reply:
I will return the fetch, if I can figure out a way to test the handleMethod.